scripts/install-template.js

```diff
@@ -0,0 +1,102 @@
+const fs = require('fs');
+const path = require('path');
+const shelljs = require('shelljs');
+
+const matchingPackages = /^@commercetools-frontend\/(?!ui-kit)(.*)/;
+const applicationName = 'my-starter-app';
+const rootPath = path.join(__dirname, '..');
+const tarballsDistPath = path.join(rootPath, 'dist-tarballs');
+const sandboxPath = path.join(rootPath, '..', 'sandbox-installation-template');
+const binaryPath = path.join(sandboxPath, 'node_modules/.bin/create-mc-app');
+const applicationPath = path.join(sandboxPath, applicationName);
+
+const templateName = process.env.TEMPLATE_NAME;
+if (!templateName) {
+  throw new Error('Missing required environment variable "TEMPLATE_NAME"');
+}
+
+if (!fs.existsSync(tarballsDistPath)) {
+  throw new Error(
+    'The dist-tarballs folder does not exist. Please run the build-tarballs.js script before this.'
+  );
+}
+
+const branchName =
+  process.env.GITHUB_EVENT_NAME === 'pull_request'
+    ? process.env.GITHUB_HEAD_REF
+    : 'master';
+
+console.log('Cleaning up sandbox folder');
+shelljs.rm('-rf', sandboxPath);
+shelljs.mkdir('-p', sandboxPath);
+
+const createMcAppTarballName = shelljs
+  .ls(tarballsDistPath)
+  .find(tarballName => tarballName.includes('create-mc-app'));
+if (!createMcAppTarballName) {
+  throw new Error('Failed to find tarball for create-mc-app package');
+}
+
+console.log(`Installing create-mc-app from tarball`);
+shelljs.exec(
+  [
+    'yarn add',
+    `file:${path.join(tarballsDistPath, createMcAppTarballName)}`,
+    '--no-lockfile',
+  ].join(' '),
+  { cwd: sandboxPath, stdio: 'ignore' }
+);
+
+console.log(
+  `Using package binary to install the application ${applicationName}, using the template ${templateName}`
+);
+shelljs.exec(
+  [
+    binaryPath,
+    applicationName,
+    '--template=starter',
+    `--template-version=${branchName}`,
+    // Don't install deps automatically, we need to patch the package.json to use the
+    // local tarballs for resolving the dependencies.
+    '--skip-install',
+  ].join(' '),
+  { cwd: sandboxPath }
+);
+
+console.log(
+  'Patching the application package.json to use local imports for appkit dependencies'
```

Comment:
Here we rewrite the appkit dependencies defined in the `package.json` by using the `file:/path/to/tarball` instead of the npm version. This way we make sure that the "latest" changes are tested.

Reply:
Ah nice!